widget/entry.go

```diff
@@ -892,6 +925,12 @@ func (e *Entry) registerShortcut() {
 	})
 }
 
+func (e *Entry) requestFocus() {
+	if c := fyne.CurrentApp().Driver().CanvasForObject(e.super()); c != nil {
+		c.Focus(e.super().(fyne.Focusable))
```

Comment:
Shouldn't is it better to get first the super() in a variable to avoid having to deal internal locks inside BaseWidget?
```go
impl := e.super()
...
c.Focus(impl.(fyne.Focusable))
```